### PR TITLE
Fix missing migrations check

### PR DESCRIPTION
**Pull request by rubickcz**

### CODE REVIEW

File: developers_chamber/qa/checks.py [hunk 1]
```diff
@@ -17,7 +19,10 @@ def _run_check(self):
         )
         last_line = output.split('\n')[-1]
         if last_line != 'No changes detected':
-            raise QAError('Found missing migration(s)!', re.findall(r'\x1b.*?Migrations.+$', output, re.DOTALL)[0])
```
Comment (rubickcz) [inline developers_chamber/qa/checks.py:20]: This regexp relied on ANSI escape sequences for color definitions. If output did not for any reason contain these sequences, the regexp failed to match. To ensure maximum compatibility, I am stripping these color sequences out.

File: developers_chamber/qa/checks.py [hunk 2]
```diff
@@ -94,3 +99,21 @@ def _run_check(self):
...
+    """
+    Checks that test methods are named correctly (i.e. does not begin with "test_should").
+    """
+    name = 'Check test method names'
+
+    def _run_check(self):
+        disallowed_method_names = []
+        for diff_obj in self._get_diffs():
+            for line in diff_obj.diff.decode().split('\n'):
+                match = re.search(r'^\+ +def (test_should_[^\(]+)\(', line)
```
Comment (rubickcz) [inline developers_chamber/qa/checks.py:114]: @radimsuckr pointed out, that this regexp should be project configurable. I sorta agree. What do you think, @matllubos?

File: developers_chamber/qa/checks.py [hunk 3]
```diff
@@ -94,3 +99,21 @@ def _run_check(self):
         wrong_import_order_files = set(changed_files) & set([diff.b_path for diff in self._get_unstaged()])
         if wrong_import_order_files:
             raise QAError('Found unsorted import(s) in following file(s):', '\n'.join(wrong_import_order_files))
+
+
+class TestMethodNamesQACheck(QACheck):
```
Comment (rubickcz) [inline developers_chamber/qa/checks.py:104]: This and code related to this was cherry picked, as it was accidentally removed in this commit: https://github.com/druids/developers-chamber/pull/69/commits/fb1be5672dd31acc8036f415584214aa81074ffc

File: developers_chamber/qa/checks.py (hunk 2 above)
Comment (matllubos) [reply developers_chamber/qa/checks.py:114]: Yes this is good idea

File: developers_chamber/qa/checks.py (hunk 3 above)
Comment (rubickcz) [reply developers_chamber/qa/checks.py:104]: I've only added the `QA_DISALLOWED_TEST_METHOD_NAME_REGEXP` setting.

**Pull request merged**